spec/spec_helper.rb

```diff
@@ -27,17 +27,17 @@
 require "rspec/longrun"
 #require "narray/narray"
 
-require "../lib/nmatrix"
-require "../lib/nmatrix/rspec"
+require "./lib/nmatrix"
+require "./lib/nmatrix/rspec"
```

Comment:
I found the commit https://github.com/prasunanand/nmatrix/commit/6d50c1f53d734009f5d3bf571e920aa17c50504e. More than 1 file is changed.  If I revert all changes are lost. What should I do?